checkout-and-start.sh

```diff
@@ -26,5 +26,12 @@ fi
 
 mvn --no-transfer-progress -V -pl war --also-make clean package -DskipTests -Dmaven.test.skip=true -P quick-build
 
+if [[ ${JENKINS_HOME:-} != "" ]]; then
```

Comment:
Simpler and more idiomatic:

```suggestion
if [[ -n ${JENKINS_HOME-} ]]; then
```

See [Bash Conditional Expressions](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html). This is analogous to `str.isEmpty()` vs `!str.equals("")` in Java.